main.py

```diff
@@ -0,0 +1,33 @@
+﻿from flask import Flask, request, json
+from PIL import Image, ImageDraw
+
+
+app = Flask(__name__)
+
+
+@app.route("/", methods=['GET', 'POST'])
+def make_matrix():
+    a = request.json
+    dict = {'Not_Available': 'red', 'Available': 'green', 'Cashbox': 'yellow', 'Rack': 'blue'}
```

Comment:
1) `dict` - reserved python identifier, choose other name,
2) could you please move states dictionary into another place, more global?